### ISSUE TITLE
project setup issue

### ISSUE THREAD

**Issue by HubertTatar** [vertica/dbt-vertica#47]
Hey

Trying to set up a project on Ubuntu 20.04.
While initializing the project I get `No adapters available`

```mkdir vertica 
cd vertica
virtualenv venv
source ./venv/bin/activate
pip install dbt-vertica

dbt init research 
Running with dbt=1.3.1
No adapters available. Go to https://docs.getdbt.com/docs/available-adapters


pip --version
pip 22.3 from /home/.../venv/lib/python3.8/site-packages/pip (python 3.8)
```

Any suggestions ?

**Comment by andyreagan** [comment]
Yeah I can reproduce this. I hadn't know about the `dbt init` command before.

Perhaps we need to add a profile template for the adapter. https://docs.getdbt.com/reference/commands/init#profile_templateyml

@HubertTatar you can use this adapter, but I suppose you can't scaffold the project with `dbt-init` just yet.

**Comment by Sachin-Thakur** [comment]
We have added dbt --init feature but not yet been released once the dbt-vertica==1.3.0 version is released this feature will be coming in until then profile information should be added manually

**Comment by andyreagan** [comment]
Thanks @Sachin-Thakur
